### PR TITLE
Make it possible to disable structured publish

### LINKED ISSUE #66: Make it possible to deactivate structure publish
With a recent change in `pulp_deb` the structured publish is now activated by default. https://github.com/pulp/pulp_deb/pull/859

However the current implementation of the `publication` subcommand only allows for the `--structured` flag to be set to `True` and thus it is impossible to deactivate structured publish when using the CLI.

### PR DESCRIPTION
**Pull request by hstct**
closes #66 

In regards to the change here: https://github.com/pulp/pulp_deb/pull/859

### CODE REVIEW

**Comment by hstct** [comment]
Ideally we could change the flag from `--structured` to `--no-structured` but it probably needs some more work since just changing the flag name will not work :)

**Comment by hstct** [comment]
> Ideally we could change the flag from --structured to --no-structured but it probably needs some more work since just changing the flag name will not work :)

Well, turns out it was pretty straight forward.

File: pulpcore/cli/deb/publication.py [hunk 1]
```diff
@@ -63,10 +63,10 @@ def publication(ctx: click.Context, pulp_ctx: PulpCLIContext, publication_type:
         help=_("Apt only: Activate simple publishing mode"),
     ),
     click.option(
-        "--structured",
+        "--structured/--no-structured",
         is_flag=True,
```
Comment (mdellweg) [inline pulpcore/cli/deb/publication.py:67]: FTR: With this option name, click implies `is_flag`.

File: pulpcore/cli/deb/publication.py [hunk 2]
```diff
@@ -63,10 +63,9 @@ def publication(ctx: click.Context, pulp_ctx: PulpCLIContext, publication_type:
         help=_("Apt only: Activate simple publishing mode"),
     ),
     click.option(
-        "--structured",
-        is_flag=True,
+        "--structured/--no-structured",
         default=None,
-        help=_("Apt only: Activate structured publishing mode"),
+        help=_("Apt only: Whether or not activate structured publishing mode"),
```
Comment (quba42) [inline pulpcore/cli/deb/publication.py:68]: Grammer nitpick:
```suggestion
        help=_("Apt only: Whether or not to activate structured publishing mode"),
```

**Review by quba42 — APPROVED** [review]
Apart from my final help output nitpick above, this is good to go.

**Review by quba42 — APPROVED** [review]
LGTM. Up to you if you want to add a changelog entry for this. I am fine either way.

**Review by quba42 — APPROVED** [review]

**Pull request merged**